### ISSUE TITLE
🔧 [CHANGE] - Add The Tour to the top level menu

### ISSUE THREAD

**Issue by RubenOlsen** [actualbudget/docs#469]
### Description

From https://discord.com/channels/937901803608096828/1118259961529634836/1271166146548207686

I suggest to add The Tour to the top level menu as this is more akin to marketing material than actual documentation of AB.

### Expected behavior

PowerAWBS — 07/10/2024 22:44
I think this is a great idea; this is like an actual version of "schedule a demo"

**Comment by Jonathan-Fang** [comment]
I agree! We just need to figure out what it's going to exactly look like and the wording.
